File: 2012/submissions/Pat Smuk/1/Ruby/Problem1.rb
#
# A Ruby solution to problem 1.
# By Pat Smuk.
# Tested with Ruby 1.9.3 on Linux Mint
#

print "Enter the word to search for: "
search = gets.chomp()
print "Enter the text to search: "
text = gets.chomp()

puts "First index of \"%s\" in text: %d" % [search, text.index(search)]

File: 2012/submissions/Pat Smuk/1/Perl/Problem1.pl
#!/usr/bin/perl
use strict;
use warnings;

# Problem 1 implemented in Perl.
# By Pat Smuk

my $word;
my $text;
my $index;

print "Enter the word to search for: ";
chomp($word = <>);

print "Enter the text to search: ";
chomp($text = <>);

$index = index $text, $word;
print "First index of \"". $word ."\" in text: ". $index ."\n";